services/app-api/handlers/reports/fetch.ts

```diff
@@ -63,7 +67,7 @@ export const fetchReport = handler(async (event, _context) => {
       Key: `${buckets.FIELD_DATA}/${state}/${fieldDataId}.json`,
     };
 
-    const fieldData = await s3Lib.get(fieldDataParams); // TODO: strict typing
+    const fieldData = (await s3Lib.get(fieldDataParams)) as AnyObject; // TODO: strict typing
```

Comment:
Comment: Due to the nature of `fieldData`, I don't think you're ever going to be able to do better than `AnyObject` for a type. It is by definition any object, with any number of key:value pairs containing anything.